conda/recipes/mantiddocs/meta.yaml

```diff
@@ -36,6 +36,9 @@ requirements:
     - sphinx_bootstrap_theme {{ sphinx_bootstrap_theme }}
     - mantidqt {{ version }}
     - versioningit {{ versioningit }}
+    - graphviz {{ graphviz }}
```

Comment:
What of these are still needed?

Reply:
Currently we install LaTeX in our linux docker image because texlive_core was missing at least one package that we required (see [here](https://github.com/mantidproject/dockerfiles/blob/28beddf097bae984e23e83f8c3487bb370737056/Linux/development/docker/CentOS7Slim.Dockerfile#L41)).
If texlive-core now works for us that would be great.

Reply:
I don't think we should be running latex anymore at all. That was used for generating the equations in the static HTML that got compiled for qtassistant. The dynamic pages are using mathjax (javascript).